packages/integration_test/example/pubspec.yaml

```diff
@@ -3,7 +3,7 @@ description: Demonstrates how to use the integration_test plugin.
 publish_to: 'none'
 
 environment:
-  sdk: ">=2.1.0 <3.0.0"
+  sdk: '>=2.10.0-56.0.dev <3.0.0'
   flutter: ">=1.6.7 <2.0.0"
```

Comment:
Should we also update flutter version?

Reply:
I think the Dart constraint will take care of that, but we could.